src/debugger_requests.jl

```diff
@@ -695,11 +695,26 @@ end
 function setp_in_request(conn, state::DebuggerState, params::StepInArguments)
     @debug "stepin_request"
 
-    put!(state.next_cmd, (cmd = :stepIn,))
+    put!(state.next_cmd, (cmd = :stepIn, targetId = params.targetId))
 
     return nothing # StepInResponseArguments()
 end
 
+function step_in_targets_request(conn, state::DebuggerState, params::StepInTargetsArguments)
+    @debug "stepin_targets_request"
+
+    targets = []
+    try
+        targets = calls_on_line(state)
+    catch err
+        @error exception = err, catch_backtrace()
```

Comment:
Shouldn't we just crash here, i.e. not have a try catch at all? So that crash reporting will tell us about it?

Reply:
Ah, yeah. This is just a remnant of my local debugging (because otherwise the REPL crashes and I can't look at the backtrace).